tests/snippets/set.py

```diff
@@ -29,23 +29,23 @@
 
 assert (set() == []) is False
 assert set().__eq__([]) == NotImplemented
-assert_raises(TypeError, lambda: set() < [], "'<' not supported between instances of 'set' and 'list'")
-assert_raises(TypeError, lambda: set() <= [], "'<=' not supported between instances of 'set' and 'list'")
-assert_raises(TypeError, lambda: set() > [], "'>' not supported between instances of 'set' and 'list'")
-assert_raises(TypeError, lambda: set() >= [], "'>=' not supported between instances of 'set' and 'list'")
+assert_raises(TypeError, lambda: set() < [])  # '<' not supported between instances of 'set' and 'list'
```

Comment:
Would it be possible to retain the error message? This could be useful when the test fails.

Reply:
```
Traceback (most recent call last):
  File "tests/snippets/set.py", line 32, in <module>
    assert_raises(TypeError, lambda: set() < set())  # '<' not supported between instances of 'set' and 'list'
  File "/Users/youknowone/Projects/RustPython/tests/snippets/testutils.py", line 8, in assert_raises
    f(*f_args, **kw)
  File "/Users/youknowone/Projects/RustPython/tests/snippets/testutils.py", line 22, in __exit__
    assert False, failmsg
AssertionError: TypeError was not raised
```

This is the result when it has error. The message is not printed at last line but still readable on the traceback as a part of code.

It will be still possible to keep the original behavior with a keyword argument. Do you prefer that way?

Reply:
Having the error message along with the error, and not in the traceback is beneficial. It's not super important to me however.

Reply:
Please check the new commit is what you wanted